src/services/corporateMatching.service.ts

```
@@ -0,0 +1,36 @@
+    try {
+      // In production, hit Double the Donation API. Returning mock verification for demonstration:
+      if (!employerName) return { eligible: false };
+      
+      const response = await axios.get(`${this.BASE_URL}/match`, {
+        params: { api_key: this.API_KEY, company: employerName },
+        timeout: 5000,
+      });
+
+      return {
+        eligible: response.data?.matching_gift_eligible || true,
+        guidelines: response.data?.guidelines || '1:1 match ratio up to $1,000.',
```

ESLint

[error] Replace `'1:1·match·ratio·up·to·$1,000.'` with `"1:1·match·ratio·up·to·$1,000."`

(prettier/prettier)